### ISSUE TITLE
Cause error if I do don't add the cacheDir variable.

### ISSUE THREAD

**Issue by qq253498229** [nwutils/nw-builder#807]
### Issue Type

- [x] Bug Fix
- [ ] Feature
- [ ] Other

### Current/Missing Behaviour

When I execute the nwbuild command, if I don't add the cacheDir variable, an exception will thrown:

```
$ nwbuild ./dist/* --mode=build --version=0.73.0 --flavor=normal --platform=win --arch=x64 --outDir=./out
[ DEBUG ] 2023-03-01T03:14:53.777Z nwbuild property is not defined in package.json
[ ERROR ] 2023-03-01T03:14:53.780Z The "path" argument must be of type string. Received undefined
[ ERROR ] 2023-03-01T03:14:53.780Z The "path" argument must be of type string. Received undefined
node:internal/errors:490
    ErrorCaptureStackTrace(err);
    ^

TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received undefined
    at new NodeError (node:internal/errors:399:5)
    at validateString (node:internal/validators:163:11)
    at resolve (node:path:1098:7)
    at parse (file:///Users/wangbin/src/demo-types/nwjs/demo-nwjs1/build/node_modules/nw-builder/src/util/parse.js:26:22)
    at nwbuild (file:///Users/wangbin/src/demo-types/nwjs/demo-nwjs1/build/node_modules/nw-builder/src/nwbuild.js:96:21) {
  code: 'ERR_INVALID_ARG_TYPE',
  level: 'error',
  timestamp: '2023-03-01T03:14:53.780Z',
  [Symbol(level)]: 'error',
  [Symbol(message)]: '[ ERROR ] 2023-03-01T03:14:53.780Z The "path" argument must be of type string. Received undefined'
}

Node.js v18.14.2
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```
### Expected/Proposed Behaviour

Normal build application

### Additional Info

- Operating System: MacOS 13.2 (22D49)
- Node version: v18.14.2
- NW.js version: 0.73.0
- Repro link: 
- ...

**Comment by tharatau** [comment]
Thanks for the bug report, I was able to reproduce.

**Comment by tharatau** [comment]
The fix is available at `v4.1.0-beta.1`!
